### PR TITLE
Adds a Rice Hat

### PR DESCRIPTION
**Pull request by SconesC**
test

For many years, famers have worn this reliable hat to keep the sun away. Now you too can be a properly attired while you prepare your crop for the Emperor's tax collectors

Huge thanks to @optimumtact for helping me figure git stuff out

### CODE REVIEW

File: code/modules/clothing/head/misc.dm [hunk 1]
```diff
@@ -228,3 +228,9 @@
 	name = "jester hat"
 	desc = "A hat with bells, to add some merryness to the suit."
 	icon_state = "jester_hat"
+
+/obj/item/clothing/head/rice_hat
+	name = "rice hat"
+	desc = "Welcome to the rice fields, motherfucker."
```
Comment (Incoming5643) [inline code/modules/clothing/head/misc.dm]: I was going to make the joke, but you beat me to it.

**Comment by WJohn** [comment]
:+1: on the sprites.

File: code/modules/hydroponics/biogenerator.dm [hunk 2]
```diff
@@ -305,6 +306,9 @@
 		if("overcoat")
 			if (check_cost(1000/efficiency)) return 0
 			else new/obj/item/clothing/suit/jacket/leather/overcoat(src.loc)
+		if("rice_hat")
+			if (check_cost(300/efficiency)) return 0
+			else new/obj/item/clothing/head/rice_hat(src.loc)
 		//if("monkey")
 		//	if (check_cost(500)) return 0
 		//	else new/mob/living/carbon/monkey(src.loc)
```
Comment (Iamgoofball) [inline code/modules/hydroponics/biogenerator.dm]: i think this was accidentally commented out

please fix

Comment (SconesC) [inline code/modules/hydroponics/biogenerator.dm]: Nope it was like that when I found it, oddly enough. Check Biogenerator.dm

Comment (Iamgoofball) [inline code/modules/hydroponics/biogenerator.dm]: the joke

Comment (Iamgoofball) [inline code/modules/hydroponics/biogenerator.dm]: .

Comment (Iamgoofball) [inline code/modules/hydroponics/biogenerator.dm]: your head

Comment (SconesC) [inline code/modules/hydroponics/biogenerator.dm]: die!

**Comment by freerealestate** [comment]
what inspired this?

**Comment by SconesC** [comment]
I was growing rice in Botany and I realized

I need a hat

**Comment by PKPenguin321** [comment]
welcome to the rice fields motherfucker

**Comment by SconesC** [comment]
haha merge conflicts i cant seem to resolve end my life now

**Comment by optimumtact** [comment]
you're clearly not properly updating one of the dmi's so it's always conflicting kek

**Comment by RandomMarine** [comment]
If only it was pathed from head/wizard/ so wizards can take lightning and pretend to be Raiden.

**Comment by Miauw62** [comment]
welcome to space station thirteen, after five years in development, hopefully it will have been worth the weight.

my favorite job is the hos. thanks, and have fun

**Comment by daman997** [comment]
ill give you 3 tasers for that unusual flaming rice hat

**Comment by SconesC** [comment]
Well despite my best attempts to update it just won't

head.dmi should be up to date

**Comment by SconesC** [comment]
I give up

**Comment by optimumtact** [comment]
Merge conflicts SconesC, are you making sure you update master to the latest upstream/master before doing the merge conflict fixes?

**Comment by Antimattercarp** [comment]
Are Black and white PJs in the works?

**Pull request closed without merging**